README.md

```diff
@@ -68,6 +68,41 @@ For this study, the Landsat 8 images for the same regions were used for which le
 
 <img src="misc/landsat8/ndwi_lmc_okeechobee.jpg" width="400">
 
+## Algorithm of performing the Thresholding on Sentinel2A indices 
+
+- Converted the TIFF file corrosponding to the interested intex into a gray scale image
+
+- Plotted the graph of number of pixels Vs. gray scale to find the value of the threshold
+
+- Generated the binary image with respect to the threshold obtained in the prevoius step
+
+## Results obtained by the implementation of the algorithm
+
+- NDWI(leaf moisture content) for Chilikha, Orissa, India
+
+<img src="misc/thresholding_sentinel2A/chilikha_ndwi_lmc_t144.PNG" width="400">
```

Comment:
Image is not showing up in the readme. Same comment for the other four images. Please check.

Reply:
@Chintan2108 I have fixed the issue with the readme.